docs/Using-Fleet/Learn-how-to-use-Fleet.md

```diff
@@ -39,15 +39,15 @@ This question can easily be answered by running this simple query: "Get operatin
 To run this query on your device:
 
 1. Select **Queries** in the top navigation.
-2. Enter "Get operating system information" in the search bar.
-3. Select **Get operating system information** to enter the **query console**.
+2. Select **Create new query** (or browse your organization's queries for "operating system information" in the search bar).
```

Comment:
This was using search method for premade queries found only in dogfood. I replaced the verbiage with run `SELECT * FROM os_verison`. But should we remove the whole idea of "search for the query" from this doc?